### PR TITLE
New package: python3-dbus-next-0.2.3

### LINKED ISSUE #46495: Package request: dbus-next
### Package name

dbus-next

### Package homepage

https://python-dbus-next.readthedocs.io/en/latest/index.html

### Description

This lib is needed for Qtile working
https://docs.qtile.org/en/stable/manual/install/index.html

After last update i cant install this package by pip (PEP 668)

### Does the requested package meet the package requirements?

System

### Is the requested package released?

Yes

### PR DESCRIPTION
**Pull request by ftpd**
#### Testing the changes
- I tested the changes in this PR: **YES**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x86_64.
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - armv6l

closes #46495

### CODE REVIEW

File: srcpkgs/python3-dbus-next/template [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+version=0.2.3
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3"
+short_desc="Next great DBus library for Python"
+maintainer="Bartek Stalewski <ftpd@insomniac.pl>"
+license="MIT"
+homepage="https://github.com/altdesktop/python-dbus-next"
+changelog="https://github.com/altdesktop/python-dbus-next/blob/master/CHANGELOG.md"
```
Comment (classabbyamp) [inline srcpkgs/python3-dbus-next/template:12]:
```suggestion
changelog="https://github.com/altdesktop/python-dbus-next/raw/master/CHANGELOG.md"
```

File: srcpkgs/python3-dbus-next/template [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Template file for 'python3-dbus-next'
+pkgname=python3-dbus-next
+version=0.2.3
+revision=1
+build_style=python3-module
+hostmakedepends="python3-setuptools"
+depends="python3"
+short_desc="Next great DBus library for Python"
```
Comment (classabbyamp) [inline srcpkgs/python3-dbus-next/template:8]:
```suggestion
short_desc="Next great DBus library"
```
self-evident from the package name

File: srcpkgs/python3-dbus-next/template (hunk 1 above)
Comment (ftpd) [reply srcpkgs/python3-dbus-next/template]: Done.

File: srcpkgs/python3-dbus-next/template (hunk 2 above)
Comment (ftpd) [reply srcpkgs/python3-dbus-next/template]: I've taken it directly from upstream's README.md ;-) Fixed.

**Pull request merged**